### PR TITLE
[1530403] Improve etcd group error message

### PR DESCRIPTION
**Pull request by mtnbikenc**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1530403

### CODE REVIEW

File: playbooks/init/evaluate_groups.yml [hunk 1]
```diff
@@ -45,7 +45,11 @@
   - name: Evaluate groups - Fail if no etcd hosts group is defined
     fail:
       msg: >
-        Running etcd as an embedded service is no longer supported.
+        Running etcd as an embedded service is no longer supported. If this is a
+        new install please define an 'etcd' group with either one, three or five
+        hosts. These hosts may be the same hosts as your masters. If this is an
+        upgrade you may set openshift_master_unsupported_embedded_etcd=true
+        until a migration playbook becomes available.
```
Comment (sdodson) [inline playbooks/init/evaluate_groups.yml]: Can you alter the last sentence to be something along the lines of `If this is an upgrade please see https://docs.openshift.com/container-platform/latest/install_config/upgrading/migrating_embedded_etcd.html for documentation on how to migrate from embedded to external etcd.`

**Comment by mtnbikenc** [comment]
Error message updated as requested.

**Pull request merged**